### ISSUE TITLE
Question about a critical feature

### ISSUE THREAD

**Issue by Fenlow** [mpv-player/mpv#12542]
At some point someone added a Christian holiday apparel item to the UI of this program, which I do not want to discuss but is it possible to completely remove every evidence of it easily without re-compiling?

**Comment by Dudemanguy** [comment]
`--script-opts=osc-greenandgrumpy=yes`

**Comment by Fenlow** [comment]
the code is still there sadly.. i wondered if its possible to fully strip it, which is helpful during inspections

**Comment by avih** [comment]
Yes, it's very much possible to fully strip it, but we won't do that.

You can do that in your copy of the source code though.
